examples/record/sketch.js

```diff
@@ -7,17 +7,14 @@ var mic, recorder, soundFile;
 var state = 0; // mousePress will increment from Record, to Stop, to Play
```

Comment:
Converting vars to let would be a good improvement in code style.
Also the variable name state =0 is not so semantic about what it is doing ,
Maybe a string like 
let recordingStarted = true/false 
Would make sense or smth like that

Reply:
Agree with the vars to let part

But regarding the renaming of state, we need to cover a total of 3 cases. Hence, it cannot be bool.
We can overcome this in two ways
1. we can maintain 2 booleans like isRecordingStarted and isResultPlayed and use them both to get 3 cases
2. We can use something like a string, but we should set the value of the string after each conditional and use it as a condition for next step

I think 2nd case would not be better than the current situation.It would be better if we change it to case 2, which can be debatable
Please share your thoughts/opinions on this :)

Reply:
I think using multiple books would be good instead of using string .
Let's just keep in sleak and intuitive with bools 
:)

Reply:
Hey I've implemented the 2 bools in a separate commit.
If you like that approach better, you can include while merging or else you can leave it out